packages/component-library/src/SimpleLegend/SimpleLegend.js

```diff
@@ -9,11 +9,15 @@ const SimpleLegend = ({ legendData }) => {
     font-weight: bold;
     text-align: center;
     margin: 10px 0 0 0;
+
+    @media (max-width: 640px) {
+        text-align: left;
+    }
   `;
 
   if (legendData.length) {
     return (
-      <legend className={legendStyle}>
+      <div className={legendStyle}>
```

Comment:
I imagine there's a default property set for the legend element in the safari user agent styles that could be overridden to make this consistent, but I don't mind the switch to a `div` either.